File: app/src/main/java/com/canopas/campose/kotlinflow_openweather/data/fivedayweather/Sys.java
package com.canopas.campose.kotlinflow_openweather.data.fivedayweather;

import com.google.gson.annotations.SerializedName;

public class Sys {

  @SerializedName("pod")
  private String pod;

  public String getPod() {
    return pod;
  }

  public void setPod(String pod) {
    this.pod = pod;
  }
}

File: app/src/main/java/com/canopas/campose/kotlinflow_openweather/data/Sys.kt
package com.canopas.campose.kotlinflow_openweather.data

import com.google.gson.annotations.SerializedName

class Sys {
    @SerializedName("country")
    var country: String? = null

    @SerializedName("sunrise")
    var sunrise = 0

    @SerializedName("sunset")
    var sunset = 0

    @SerializedName("message")
    var message = 0.0
}